domains/mtestsite.json

```diff
@@ -1,11 +1,10 @@
- 
-    {
-      "owner": {
-        "username": "mobin-mazahery-school",
-        "email": "mobin.mazahery85@gmail.com"
-      },
-      "record": {
-        "CNAME": "m-proxy.cf"
-      }
+{
+        "owner": {
+           "username": "mobin-mazahery-school",
+           "email": "mobin.mazahery.school@gmail.com"
+        },
+        "record": {
+              "TXT": "hello"
```

Comment:
TXT record alone doesn't do anything.

Reply:
I used the manage.is-a.dev and didn't made the changes by hand. there should be a problem with the panel